packages/catalog-search/src/SearchSuggestions.jsx

```diff
@@ -57,10 +57,10 @@ const SearchSuggestions = ({
       </button>
     </div>
   );
-};
+}
 
 SearchSuggestions.propTypes = {
-  autoCompleteHits: PropTypes.arrayOf(PropTypes.object).isRequired,
+  autoCompleteHits: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
```

Comment:
nit: maybe we can include content_type here?

Reply:
Bump.